README.md

```diff
@@ -50,11 +50,30 @@ Copy the `dist/omg.js` file into your project and include it.
 
 ### Examples
 
-You can find example code inside `examples` folder
+You can find example code inside `examples` folder. To run the examples, step inside the `examples` directory. Run `npm install`. Edit your `config.js` file to have to correct values.
 
-[Deposit ETH through Rootchain Contract](examples/node-deposit.js)
+_Note: You may need to also run `npm install` in the `packages\omg-js-rootchain`, `packages\omg-js-childchain` and `packages\omg-js-util` directories._
```

Comment:
hmmm does the npm install on the root directory install all the deps in sub-packages automatically ? (at least I believed this was how Lerna works)

Reply:
I've never used Lerna, but npm requires it to be done manually, or writing a `postinstall` script in the base package that cds into all the subdirectories and runs npm install. This snagged for a minute, so that's why I added it as comment.

Reply:
Ah, I understand your comment about Lerna now. I've remove my 'Note' from the README as it's unnecessary.